proposed/storageos-operator/v0.3.0/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+appVersion: "v2.1.0"
+description: Cloud Native storage for containers
+name: storageos-operator
+version: 0.3.0
+tillerVersion: ">=2.10.0"
+keywords:
+- storage
+- block-storage
+- volume
+- operator
+home: https://storageos.com
+icon: https://docs.storageos.com/images/logo-helm.svg
```

Comment:
Please make a local copy of the logo in the parent folder `storageos-operator/` and reference the image here as `file://../logo-helm.svg`. This way the logo can be displayed in airtight environments.

Reply:
Thanks. Added.